languages/csharp/config.json

```diff
@@ -24,6 +24,12 @@
         "concepts": ["booleans"],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "casting",
+        "uuid": "0d63645d-385a-43ea-b89e-448ed608f6ce",
+        "concepts": ["casting"],
+        "prerequisites": ["exceptions", "inheritance", "numbers"]
```

Comment:
For future reference: we should probably have the `numbers` exercise also have a `type-conversion` concept.

Reply:
raised issue #1863